pkg/apis/serving/v1beta1/predictor_custom.go

```diff
@@ -73,3 +76,8 @@ func (c *CustomPredictor) IsMMS(config *InferenceServicesConfig) bool {
 	//TODO: Figure out how to check if custom predictor is mms
 	return false
 }
+
+func (c *CustomPredictor) IsFrameworkSupported(framework string, config *InferenceServicesConfig) bool {
+	//TODO: Figure out how to check if custom predictor is supports framework
+	return false
```

Comment:
I think we can turn true here as custom framework can use any arbitrary model framework or code as long as isMultiModelServer flag is turned on in config.

Reply:
IsMultiModelServer returns false by default because there is no config for predictor custom so IsFrameworkSupported would always return false. I am good with setting the default return to true though.